README.md

```diff
@@ -147,9 +150,9 @@ It is called "stat size" because it's obtained from Webpack's
 This is the "output" size of your files. If you're using a Webpack plugin such
 as Uglify, then this value will reflect the minified size of your code.
 
-### `gzip`
+### `compressed`
```

Comment:
Can we make it so that the added option for `defaultSizes` would be `brotli` instead of `compressed`? That way the current, default gzip usage would not have to be changed.

That way we could keep the gzip-related documentation intact and only worry about adding new documentation related to the brotli options.

I could imagine that the fourth size definition documentation here could look something like this:

> ### `brotli`
> 
> This is the size of running the parsed bundles/modules through [Brotli](https://www.brotli.org/) compression. **Note:** This needs the compression algorithm to be configured to use Brotli.